packages/desktop/src/main.rs

```diff
@@ -22,8 +22,6 @@ use tokio::{
 use tracing::{info, level_filters::LevelFilter};
 use tracing_subscriber::EnvFilter;
 
-use crate::util::window_ext::WindowExt;
-
```

Comment:
could you add this back? it's unused on windows, but it's actually used on mac. it could be changed to:

```rust
  #[cfg(target_os = "macos")]
  use crate::util::window_ext::WindowExt;
  ```

Reply:
Addressed this in the latest commit